department-of-veterans-affairs/next-build
# head changes

## Description
Relates to #[15510](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15510). 

Changes
Adds various elements to the Head component in the _document file
Adds some icon and image files to next-build public/img directory


## Testing done
local testing

## Screenshots
<img width="1395" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/569ffc1e-234b-4130-a378-2d088bf9aa59">


## QA steps
Verify that existing pages still build as expected
verify that favicon now renders properly
verify that appropriate files are loaded in network tab via their respective environment



## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
